app/models/thread.php

```diff
@@ -44,9 +48,8 @@ public function write(Comment $comment)
         }
 
         $db = DB::conn();
-        $db->query('INSERT INTO comment 
-                SET thread_id = ?, username = ?, body = ?',
-                array($this->id, Session::get('username'), $comment->body));
+        $db->query('INSERT INTO comment SET thread_id = ?, username = ?, body = ?, user_id = ?',
+                array($this->id, Session::get('username'), $comment->body, Session::get('id')));
```

Comment:
I think the values of the session must be passed as parameters in the function and not called inside the model.
a model class is an object representation of the data in a table on your database :bow: